Source/Core/VideoBackends/OGL/OGLTexture.cpp

```diff
@@ -0,0 +1,241 @@
+#include "Common/GL/GLInterfaceBase.h"
```

Comment:
I meant to indicate that it's an include that should be included along with everything else, not to replace it (my bad).